README.md

```diff
@@ -21,22 +21,37 @@ saucectl configure
 
 ## Running The Examples
 
-Simply check out this repo and run the appropriate command below :rocket:
+Simply check out this repo and set `config.yml` like following :rocket:
```

Comment:
The idea of the example repo is that the user doesn't have to set anything. The examples should be ready to run in whatever way we defined them. It's probably better to refactor the entire README and re-record a new gif that showcases the potential mixed mode. That means we don't really need to talk about docker mode or sauce mode. The mode becomes secondary, since by default, everything will just run in sauce :)

Reply:
I see. I'll remove this part in README